packages/components/notifications/content-notification/index.js

```diff
@@ -0,0 +1,6 @@
+// This file exists  because we want jest to use our non-compiled code to run tests
+// if this file is missing, and you have a `module` or `main` that points to a non-existing file
+// (ie, a bundle that hasn't been built yet) then jest will fail if the bundle is not yet built.
+// all apps should export all their named exports from their root index.js
+export { default } from './src';
+export * from './src';
```

Comment:
`content-notification` code is inside a `src` folder

Reply:
The folder structure is a bit weird. Instead of `notifications/content-notification/src` I would expect `notifications/src/content-notification`.

The package is `notifications`, and each package has a `src` folder.

Reply:
I thought of it the other way around. Similar to the `packages/components/fields` or `packages/components/inputs` where the main folder holds the other smaller components which has their own `src` folders

Reply:
Yes but it's different. The `fields` folder or the `inputs` folder are not packages, they are just folders.

Here you have the `notifications` folder being the package

<img width="597" alt="image" src="https://user-images.githubusercontent.com/1110551/88680947-5598e280-d0f1-11ea-8678-d00393ac62b4.png">

Reply:
Also, we have presets for fields and inputs packages, but for notifications I don't see a need for that.

So I would just keep the `notifications` being the package (as you have it now) but with a `src` folder.
You can still keep the `src/content-notification` structure if you want.